contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -6159,11 +6162,40 @@ transformSelectIntoStmt(CreateTableAsStmt *stmt)
 			}
 			else
 			{
+				AlterTableStmt *newstmt;
+				AlterTableCmd *cmd;
+				char* original_name = NULL;
+				newstmt = NULL;
```

Comment:
Try to stick to either CamelCase or SnakeCase when naming variables or functions.